### PR TITLE
Reporting updates

### PR DESCRIPTION
**Pull request by rvansa**

### CODE REVIEW

**Comment by alanfx** [comment]
Hey @rvansa, can you provide a benchmark file that generates a percentiles chart? I haven't been able to make one yet.

**Comment by rvansa** [comment]
@alanfx just use histogram statistics, percentiles are generated from the same data.

File: reporters/reporter-default/src/main/java/org/radargun/reporting/html/LogTimeAxis.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+         if (labelBounds != null && labelBounds.getWidth()
+               + insets.getTop() + insets.getBottom() > maxHeight) {
+            maxHeight = labelBounds.getWidth()
+                  + insets.getTop() + insets.getBottom();
+         }
+      }
+      return maxHeight;
+   }
+
+   protected double findMaximumTickLabelWidth(List ticks, Graphics2D g2, Rectangle2D drawArea, boolean vertical) {
```
Comment (mcimbora) [inline reporters/reporter-default/src/main/java/org/radargun/reporting/html/LogTimeAxis.java]: findMaximumTickLabelWidth & findMaximumTickLabelHeight are identical to PercentileLogAxis.findMaximumTickLabelWidth & PercentileLogAxis.findMaximumTickLabelHeight. Perhaps they could be extracted to some other class (e.g. AxisUtils)?

**Comment by mcimbora** [comment]
@rvansa Looks fine. I'm thinking of using freemarker templates for generating HTML pages, which will provide better separation of view & model. I'll have a look at it.

**Comment by alanfx** [comment]
@rvansa I think it would be be better to have an example of the usage of this chart in one of the benchmarks provided with the PR. As I said, I haven't been able to create one myself, so I'd appreciate if you could supply one for my testing! :-)

**Comment by rvansa** [comment]
@mcimbora created ChartsHelper with those two methods
@alanfx added histogram statistics to benchmark-dist.xml, works like charm out-of-the box :-P

**Comment by mcimbora** [comment]
@rvansa Integrated, good job!

**Pull request closed without merging**